ads/_config.js

```diff
@@ -193,6 +193,8 @@ const adConfig = jsonConfiguration({
     preconnect: 'https://ad.ad-stir.com',
   },
 
+  'adstyle': {},
```

Comment:
I see your not using renderStart here, we recommend implementing it if possible, should help with UX and get some performance gains. It is optional.